src/scripts/main.js

```diff
@@ -1,3 +1,49 @@
 'use strict';
 
 // write code here
+const listEmployee = document.querySelectorAll('li');
+
+function parseSalary(salaryString) {
+  const cleanedString = salaryString.replace(/[^0-9.-]+/g, '');
+
+  // Перевіряємо, чи це дійсно валідне число
+  if (!isNaN(cleanedString) && cleanedString.trim() !== '') {
+    return parseFloat(cleanedString);
+  } else {
+    return null;
+  }
+}
+
+function sortList(list) {
+  return [...list].sort((a, b) => {
+    // Отримуємо зарплату з data-атрибутів і перетворюємо в число
+    const salaryA = parseSalary(a.dataset.salary);
+    const salaryB = parseSalary(b.dataset.salary);
+
+    // Сортуємо за спаданням зарплати
```

Comment:
Remove comment. Try to write obvious code that explains all we need without comments)